gui/app/controls/FlowGraph.qml

```diff
@@ -0,0 +1,18 @@
+import QtQuick 2.11
+import QtQuick.Controls 2.4
+import Respira 1.0
+import ".."
+
+ScopeView {
+    id: flowView
+    name: "Flow"
+    unit: "L/min"
+    // TODO: Are these reasonable lower and upper bounds?
+    // Source for current value:
+    // https://www.sciencedirect.com/topics/medicine-and-dentistry/peak-inspiratory-flow
+    // "Most modern ventilators can deliver flow rates between
+    // 60 and 120 L/min. "
+    yMin: -150
+    yMax: 150
+    dataset: GuiStateContainer.flowSeries
```

Comment:
Suggestion: Leave all assignment related to `GuiStateContainer` in the main.qml or a higher-level Item (Like in the mode itself) in case we decided to break it down. Spreading it in several files might make it mode difficult to  read and refactor later.

Reply:
This is a good suggestion too, but it seems tedious to implement without first breaking down GuiStateContainer, unless I'm missing something. Are you suggesting that e.g. CommandPressureMode should pass a "series" property to FlowGraph, and in turn main.qml should pass GuiStateContainer.flowSeries as a "flowSeries" property to CommandPressureMode?

I think this would work quite well if we split GuiStateContainer into parts like "readings", "parameters", "graphs" and main.qml passed all three of these to the mode and mode passed the relevant sub-parts to sub-components, but I'd prefer to do this separately from this PR. Added a TODO for now.